src/prognostic_equations/edmfx_tke.jl

```diff
@@ -104,38 +99,28 @@ function edmfx_tke_tendency!(
         ᶜK_h = @. lazy(eddy_diffusivity(ᶜK_u, ᶜprandtl_nvec))
 
         # shear production
-        @. Yₜ.c.sgs⁰.ρatke += 2 * ᶜρa⁰ * ᶜK_u * ᶜstrain_rate_norm
+        @. Yₜ.c.sgs⁰.ρatke += 2 * Y.c.ρ * ᶜK_u * ᶜstrain_rate_norm
```

Comment:
You'd probably want to move this out of .sgs^0 eventually. But fine to leave now if you want to make it easier to revert. Just leave a TODO.

Reply:
Yes, that's what I think too. Having said that, I don't really want to revert this. I'll run AMIP with the change before merging this, but let me know if you think of any potential issues.